### PR TITLE
Controller specs

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
In this Pull request I have done the following:

- [x] Added RSpec as a gem to your project.

- [x] Created a Request spec file for all my controllers.

- [x] Make sure that for each action you check:

1. If the response status was correct.
2. If a correct template was rendered.
3. If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/user_request_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+RSpec.describe 'Users', type: :request do
+  describe 'GET #index' do
+    it 'responds with HTTP success' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
```
Comment (DeliceLydia) [inline spec/requests/user_request_spec.rb]: - [ ] Please make sure all your tests are passing
Screenshot 2023-08-23 at 18 30 27

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
### Hi @rachaouldbabaali, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Followed git workflow
- No linter errors

## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @rachaouldbabaali 👍 👍, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
All written tests for the contoller passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rachaouldbabaali** [comment]
Thank you @OyePriscilla, I truly appreciate your time and efforts in reviewing my work

**Pull request merged**